### DISCUSSION TITLE
use_frameworks! :linkage => :static causes weird behavior

### DISCUSSION THREAD

**Discussion by espenjanson** [software-mansion/react-native-screens#1586] [Q&A] [answered]
Hi,

Thanks for all the hard work with this nice library!

When setting use_frameworks! :linkage => :static in your Podfile and using react-navigation 6 (depending on react-native-screens) and navigating somewhere -> navigating back doesn't remove the new screen (it seems). Can be fixed adding this to Podfile:

```
pre_install do |installer|
  Pod::Installer::Xcode::TargetValidator.send(:define_method, :verify_no_static_framework_transitive_dependencies) {}

  installer.pod_targets.each do |pod|
    if pod.name.eql?('RNScreens')
      def pod.build_type;
        # Uncomment the line corresponding to your CocoaPods version
        Pod::BuildType.static_library # >= 1.9
        # Pod::Target::BuildType.static_library # < 1.9
      end
    end
  end
end
```

Why is that? Could this be fixed?

Thanks!

**Comment by tboba** [comment, marked as answer]
Hi @espenjanson, thanks for reporting that! Are you still facing this issue on the latest react-native-screens version (currently 3.26.0)? If so, could you create an issue with steps to reproduce (including minimal repro)?

Thanks!
